lib/archethic/bootstrap.ex

```diff
@@ -213,7 +213,7 @@ defmodule Archethic.Bootstrap do
     SelfRepair.start_scheduler()
 
     :persistent_term.put(:archethic_up, :up)
-    Archethic.PubSub.notify_node_up(:persistent_term.get(:archethic_up, nil))
+    Archethic.PubSub.notify_node_up(:up)
```

Comment:
No need to specify the `up` information, we want just want notification it's all